Cubical/ZCohomology/Groups/Wedge.agda

```diff
@@ -45,11 +41,11 @@ module _ {ℓ ℓ'} (A : Pointed ℓ) (B : Pointed ℓ') where
     bijIso : BijectionIso (coHomGr 1 (A ⋁ B)) (×coHomGr 1 (typ A) (typ B))
     BijectionIso.map' bijIso = I.i 1
     BijectionIso.inj bijIso =
-       sElim (λ _ → isOfHLevelΠ 2 λ _ → isOfHLevelPath 2 setTruncIsSet _ _)
-              λ f inker → helper ∣ f ∣₂ (I.Ker-i⊂Im-d 0 ∣ f ∣₂ inker)
+      sElim (λ _ → isSetΠ λ _ → isProp→isSet (setTruncIsSet _ _))
+            λ f inker → helper ∣ f ∣₂ (I.Ker-i⊂Im-d 0 ∣ f ∣₂ inker)
     BijectionIso.surj bijIso =
```

Comment:
Strangely enough this is now the slowest definition in the whole library... Any ideas how to simplify it @aljungstrom ? Maybe one can prove the base case more directly without going through `BijectionIsoToGroupIso` ?

Reply:
Hmm, I think this is the weird one where I had to eliminatre truncations although I shouldn't have to. Try doing it without elimination on the first thing and maybe it'll go through this time... I think it's hard to do without the bijectionIso since MV only gives us surjective an injective functions (on a set level). But I'll think about it today